pkgs/applications/office/mmex/default.nix

```diff
@@ -1,36 +1,31 @@
 # To use this program, copy all that is in $out/opt/mmax into a writable directory,
 # and run it from there. This is the intended usage, as far as I understand.
 
-{ fetchsvn, stdenv, wxGTK }:
+{ stdenv, fetchgit, sqlite, wxGTK30, gettext }:
 
-let version = "0.9.5.1";
+let version = "v1.2.7";
 in
   stdenv.mkDerivation {
     name = "money-manager-ex-${version}";
 
-    src = fetchsvn {
-      url = "https://moneymanagerex.svn.sourceforge.net/svnroot/moneymanagerex/tags/releases/${version}";
-      sha256 = "0mby1p01fyxk5pgd7h3919q91r10zbfk16rfz1kbchqxqz87x4jq";
+    src = fetchgit {
```

Comment:
`fetchFromGithub` didn't seem to fetch submodules and I found other people online saying that. Should `fetchFromGithub` fetch submodules?